state/runtime/evm/instructions_test.go

```diff
@@ -90,3 +94,285 @@ func TestMStore(t *testing.T) {
 
 	assert.Len(t, s.memory, 1024+32)
 }
+
+type mockHostForCreate struct {
+	mockHost
+	nonce       uint64
+	callxResult *runtime.ExecutionResult
+}
+
+var _ runtime.Host = (*mockHostForCreate)(nil)
```

Comment:
What do you achieve with this line?
I don't understand 😕

Reply:
Removed it